includes/extra_configures/enable_error_logging.php

```diff
@@ -52,18 +99,30 @@ function zen_debug_error_handler ($errno, $errstr, $errfile, $errline)
 
 /**
  * The path where the debug log file will be located
- * Default value is: DIR_FS_LOGS . '/myDEBUG-999999-00000000.log'
- * ... which puts it in the /logs/ folder:   /logs/myDEBUG-999999-00000000.log  (where 999999 is a random number, and 00000000 is the server's timestamp)
+ * Default value is: DIR_FS_LOGS . '/myDEBUG-yyyymmdd-hhssmm-xxxxx.log'
+ * ... which puts it in the /logs/ folder:   /logs/myDEBUG-yyyymmdd-hhiiss-xxxxx.log
+ *     where:
+ *      - yyyy .... is the 4-digit year
+ *      - mm ...... is the 2-digit month
+ *      - dd ..... is the 2-digit day-of-month
+ *      - hh ..... is the 2-digit hour
+ *      - ii ..... is the 2-digit minute
+ *      - ss ..... is the 2-digit second
+ *      - xxxxx ... is the time in milliseconds
+ *
  *    (or if you don't have a /logs/ folder, it will use the /cache/ folder instead)
  */
-$debug_logfile_path = DIR_FS_LOGS . '/myDEBUG-' . time() . '-' . mt_rand(1000,999999) . '.log';
+$log_prefix = (IS_ADMIN_FLAG) ? '/myDEBUG-adm-' : '/myDEBUG-';
+$log_date = new DateTime();
+$debug_logfile_path = DIR_FS_LOGS . $log_prefix . $log_date->format('Ymd-His-u') . '.log';
```

Comment:
Is your use of milliseconds providing greater entropy than the 4-6-digit random number? The goal here is prevent unauthorized reading of the files.

Reply:
Since the milliseconds are, in essence, a 6-digit pseudo-random number, I believe that it's providing uniqueness.  The .htaccess in the /logs directory is also preventing that unauthorized reading.

Reply:
> The .htaccess in the /logs directory is also preventing that unauthorized reading.

Yes, on apache the .htaccess prevents access, but nginx ignores .htaccess so one can't depend on those directives.

> Since the milliseconds are, in essence, a 6-digit pseudo-random number, I believe that it's providing uniqueness.

I'll accept that for now.